easybuild/tools/run.py

```diff
@@ -86,7 +87,7 @@
 )
 
 RunShellCmdResult = namedtuple('RunShellCmdResult', ('cmd', 'exit_code', 'output', 'stderr', 'work_dir',
-                                                     'out_file', 'err_file', 'thread_id', 'task_id'))
+                                                     'out_file', 'err_file', 'cmd_sh', 'thread_id', 'task_id'))
```

Comment:
We need a docstring somewhere defining what each of these items are.

Reply:
fixed in 63ac9e56b